### ISSUE TITLE
Not working on Big Sur?

### ISSUE THREAD

**Issue by kakyoism** [peterklijn/hammerspoon-shiftit#9]
I just installed this on my Big Sur machine and unfortunately after following the instruction here, none of the shortcuts work. 

Below is my log

```sh
2021-01-30 15:42:19: Welcome to the Hammerspoon Console!
You can run any Lua code in here.

2021-01-30 15:42:19: -- Lazy extension loading enabled
2021-01-30 15:42:19: -- Loading ~/.hammerspoon/init.lua
2021-01-30 15:42:19: -- Loading Spoon: ShiftIt
2021-01-30 15:42:19: -- Loading extension: hotkey
2021-01-30 15:42:19: 15:42:19     hotkey: Enabled hotkey ⌘⌃⌥LEFT
2021-01-30 15:42:19:              hotkey: Enabled hotkey ⌘⌃⌥RIGHT
2021-01-30 15:42:19:              hotkey: Enabled hotkey ⌘⌃⌥UP
2021-01-30 15:42:19:              hotkey: Enabled hotkey ⌘⌃⌥DOWN
2021-01-30 15:42:19:              hotkey: Enabled hotkey ⌘⌃⌥1
2021-01-30 15:42:19:              hotkey: Enabled hotkey ⌘⌃⌥2
2021-01-30 15:42:19:              hotkey: Enabled hotkey ⌘⌃⌥3
2021-01-30 15:42:19:              hotkey: Enabled hotkey ⌘⌃⌥4
2021-01-30 15:42:19:              hotkey: Enabled hotkey ⌘⌃⌥M
2021-01-30 15:42:19:              hotkey: Enabled hotkey ⌘⌃⌥F
2021-01-30 15:42:19:              hotkey: Enabled hotkey ⌘⌃⌥Z
2021-01-30 15:42:19:              hotkey: Enabled hotkey ⌘⌃⌥C
2021-01-30 15:42:19:              hotkey: Enabled hotkey ⌘⌃⌥N
2021-01-30 15:42:19:              hotkey: Enabled hotkey ⌘⌃⌥P
2021-01-30 15:42:19:              hotkey: Enabled hotkey ⌘⌃⌥=
2021-01-30 15:42:19:              hotkey: Enabled hotkey ⌘⌃⌥-
2021-01-30 15:42:19: -- Done.
2021-01-30 15:43:03: -- Loading extension: window
2021-01-30 15:43:03: 15:43:03 ERROR:   LuaSkin: hs.hotkey callback: /Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: attempt to index a nil value
stack traceback:
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: in function 'move'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:113: in function 'ShiftIt.left'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:161: in function </Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:161>
2021-01-30 15:43:03: 15:43:03 ERROR:   LuaSkin: hs.hotkey callback: /Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: attempt to index a nil value
stack traceback:
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: in function 'move'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:113: in function 'ShiftIt.left'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:161: in function </Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:161>
2021-01-30 15:43:03: 15:43:03 ERROR:   LuaSkin: hs.hotkey callback: /Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: attempt to index a nil value
stack traceback:
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: in function 'move'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:115: in function 'ShiftIt.up'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:163: in function </Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:163>
2021-01-30 15:43:04: 15:43:04 ERROR:   LuaSkin: hs.hotkey callback: /Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: attempt to index a nil value
stack traceback:
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: in function 'move'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:116: in function 'ShiftIt.down'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:164: in function </Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:164>
2021-01-30 15:43:04: 15:43:04 ERROR:   LuaSkin: hs.hotkey callback: /Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: attempt to index a nil value
stack traceback:
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: in function 'move'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:116: in function 'ShiftIt.down'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:164: in function </Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:164>
2021-01-30 15:43:05: 15:43:05 ERROR:   LuaSkin: hs.hotkey callback: /Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: attempt to index a nil value
stack traceback:
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: in function 'move'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:116: in function 'ShiftIt.down'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:164: in function </Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:164>
2021-01-30 15:43:05: 15:43:05 ERROR:   LuaSkin: hs.hotkey callback: /Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: attempt to index a nil value
stack traceback:
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: in function 'move'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:114: in function 'ShiftIt.right'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:162: in function </Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:162>
2021-01-30 15:43:05: 15:43:05 ERROR:   LuaSkin: hs.hotkey callback: /Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: attempt to index a nil value
stack traceback:
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: in function 'move'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:113: in function 'ShiftIt.left'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:161: in function </Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:161>
2021-01-30 15:43:06: 15:43:06 ERROR:   LuaSkin: hs.hotkey callback: /Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: attempt to index a nil value
stack traceback:
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: in function 'move'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:114: in function 'ShiftIt.right'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:162: in function </Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:162>
2021-01-30 15:43:06: 15:43:06 ERROR:   LuaSkin: hs.hotkey callback: /Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: attempt to index a nil value
stack traceback:
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: in function 'move'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:113: in function 'ShiftIt.left'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:161: in function </Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:161>
2021-01-30 15:43:06: 15:43:06 ERROR:   LuaSkin: hs.hotkey callback: /Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: attempt to index a nil value
stack traceback:
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:51: in function 'move'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:113: in function 'ShiftIt.left'
	/Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:161: in function </Users/me/.hammerspoon/Spoons/ShiftIt.spoon/init.lua:161>

```

**Comment by peterklijn** [comment]
Hi @kakyoism,

Thanks for your issue, could you please check that you gave Hammerspoon the right permissions?

<img width="780" alt="image" src="https://user-images.githubusercontent.com/2742131/106391969-407a7480-63f0-11eb-8832-e90a682dab18.png">

I just tried it on my new machine, running:
Big Sur 11.1
Hammerspoon 0.9.82 (5614)

**Comment by marcolivierarsenault** [comment]
Had the exact same problem, Permission fixed it

**Comment by dwlf** [comment]
Restart Hammerspoon was required when I encountered a similar error after initial install.

Apple M1 Pro
macOS 12.1 (21C52) 

I had to restart Hammerspoon after giving Hammerspoon 'Accessibility' permissions for the spoon to work. Not an issue with this spoon, but this was counter intuitive as Hammerspoon set up gave the green light to proceed after the permissions had been updated.

Thank you for this awesome software! It was such a relief to find it when I discovered ShiftIt is not available for M1. The Readme.md install instructions are very good. After double clicking on the unzipped spoon, it just disappeared and Hammerspoon gave no indication that it installed anything, but I had faith ;-)
